docs/users-and-roles.md

```diff
@@ -0,0 +1,42 @@
+# Users and roles
+
+As described in our [authorization model documentation](api-authorization.md),
+we use a combination of role- and activity-based authorization. Every
+action in the system is controlled by an "activity" permission. Roles are
+essentially a collection of activities.
+
+### Roles
+
+The system has some predefined roles. New roles can be added as the system
+is running, and the predefined roles can be modified. These are the roles
+as they are defined at the "beginning":
+
+| Name               | Activities                                                                                                                  | Description                                                                                                                                                                                                    |
+| ------------------ | --------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
@mgwalker Can we expand activities column so that it formats a little nicer?

Reply:
Unfortunately, this is a function of how Github displays Markdown tables.  I'll poke around with it, but as far as I know, there's not a lot we can do to change the formatting.

Reply:
got it, no problem. I had to try and find something for my first one.

Reply:
Give it a look now, let me know what you think.  😄